fuse/dl/models/model_wrapper.py

```diff
@@ -47,6 +47,14 @@ def __init__(
         :param model: The model to wrap
         :param model_inputs: sequence of keys in batch dict to transfer into model.forward function
             for example: model_inputs=('data.input.input_0.tensor',)
+            It can also be a dictionary mapping model forward argument name to key used to extract value from batch_dict.
```

Comment:
Nice!

Although the logic is simple maybe worth adding a unit-test for it (in a future PR)